boards/posix/native_posix/Kconfig.defconfig

```diff
@@ -90,7 +90,7 @@ endif # TRACING_CTF
 
 if FLASH
 
-config FLASH_NATIVE_POSIX
+config FLASH_SIMULATOR
```

Comment:
Nit: Maybe worth mentioning in the commit msg that you also change the default and doc in this commit?

Reply:
done